### PR TITLE
i18n ru-RU

### PR DESCRIPTION
**Pull request by rockenren**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

- [ ] [Searched](https://github.com/shadowsocks/shadowsocks-windows/search?q=is%3Apr&type=Issues) for similar pull requests
- [ ] Compiled the code with Visual Studio
- [x] Require translation update
- [ ] Require document update (readme.md, wikipage, etc)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New feature

---

### Description of your *pull request* and other information

First version of Russian (ru-RU) translation.

### CODE REVIEW

File: shadowsocks-csharp/Data/i18n.csv [hunk 1]
```diff
@@ -1,207 +1,207 @@
...
+Proxy request failed,Не удалось выполнить запрос,代理请求失败,Proxy 要求失敗,プロキシ要求が失敗しました。
+Proxy handshake failed,Не удалось выполнить хэндшейк,代理握手失败,Proxy 交握失敗,プロキシ ハンドシェイクに失敗しました。
+Register hotkey failed,Не удалось применить настройки горячих клавиш,注册快捷键失败,註冊快速鍵失敗,ホットキーの登錄に失敗しました。
+Cannot parse hotkey: {0},Не удалось распознать следующие горячие клавиши: {0},解析快捷键失败: {0},剖析快速鍵失敗: {0},ホットキーを解析できません: {0}
+"Timeout is invalid, it should not exceed {0}",Таймаут не может превышать значение {0},超时无效，不应超过 {0},逾時無效，不應超過 {0},タイムアウト値が無効です。{0} 以下の値を指定して下さい。
+Cannot find the plugin program file,Файл плагина не найден,找不到插件程序文件,找不到外掛程式文件,
+,,,,
+Operation failure,Операция завершилась неудачей,操作失败,,
+Auto save failed,Автоматическое сохранение не удалось,自动保存失败,,
+Whether to discard unconfigured servers,,是否丢弃未配置的服务器,,
```
Comment (Stzx) [inline shadowsocks-csharp/Data/i18n.csv]: Forgot to translate?

Comment (rockenren) [reply shadowsocks-csharp/Data/i18n.csv]: Yes, missed that line.
Thanks for pointing me out.

**Comment by rockenren** [comment]
Added missing lines translation.

File: shadowsocks-csharp/Data/i18n.csv [hunk 2]
```diff
@@ -104,6 +104,7 @@ Ping,Ping,网络延迟,,
 Chart Mode,График,图表模式,,
 24h,24ч,24小时,,
 all,За все время,全部,,
+StatisticsStrategyConfigurationForm,Настройки статистики,,,
```
Comment (studentmain) [inline shadowsocks-csharp/Data/i18n.csv]: https://github.com/shadowsocks/shadowsocks-windows/blob/35950b9a5885154709fa66c5e4ca61a53d4a06cc/shadowsocks-csharp/Data/i18n.csv#L53

**Comment by studentmain** [comment]
I think we can merge it now.

**Pull request merged**